pkg/installer/mdsd/util.go

```diff
@@ -0,0 +1,212 @@
+package mdsd
+
+import (
+	"bytes"
+	"html/template"
+	"io"
+	"path"
+	"path/filepath"
+	"strings"
+
+	ignutil "github.com/coreos/ignition/v2/config/util"
+	igntypes "github.com/coreos/ignition/v2/config/v3_2/types"
```

Comment:
If you add these names to https://github.com/openshift/installer-aro-wrapper/blob/main/hack/validate-imports/allowed-import-names.yaml then the validate job will be happy.